File: Day 5/loops_v2.js
function processData(input) {
    var ans = "";
    var lines = input.split('\n');
    var numOfTests = parseInt(lines[0]);
    for (var i = 1; i <= numOfTests; ++i) {
        var nums = lines[i].split(' ')
        var a = parseInt(nums[0]), b = parseInt(nums[1]), N = parseInt(nums[2]);
        var currentSum = a;
        for (var j = 0; j < N; ++j) {
            currentSum += Math.pow(2, j) * b;
            ans += currentSum.toString() + " ";
        }
        ans += "\n";
    }
    process.stdout.write(ans);
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});

File: Day 5/loops_v2.cpp
#include <iostream>
#include <cmath>
#include <string>

using namespace std;

int main() {
    int numOfTests;
    cin >> numOfTests;
    int a, b, N;
    string ans;
    for (int i = 0; i < numOfTests; ++i) {
    	cin >> a >> b >> N;
    	int currentSum = a;
    	for (int j = 0; j < N; ++j) {
    		currentSum += pow(2, j) * b;
            ans += to_string(currentSum) + " ";
    	}
    	ans += (i != numOfTests - 1) ? "\n" : "";
    }
    cout << ans; 
    return 0;
}